clean_code_block/src/main/java/Airport.java

```diff
@@ -0,0 +1,124 @@
+import Planes.ExperimentalPlane;
+import models.MilitaryType;
+import Planes.MilitaryPlane;
+import Planes.PassengerPlane;
+import Planes.Plane;
+
+import java.util.*;
+
+public class Airport {
+    private List<? extends Plane> planes;
+
+    public List<PassengerPlane> getPassengerPlanes() {
+        List<? extends Plane> planeList = this.planes;
+        List<PassengerPlane> passengerPlaneList = new ArrayList<>();
+        for (Plane plane : planeList) {
+            if (plane instanceof PassengerPlane) {
+                passengerPlaneList.add((PassengerPlane) plane);
+            }
+        }
+        return passengerPlaneList;
+    }
+
+    public List<MilitaryPlane> getMilitaryPlanes() {
+        List<MilitaryPlane> militaryPlaneList = new ArrayList<>();
+        for (Plane plane : planes) {
+            if (plane instanceof MilitaryPlane) {
+                militaryPlaneList.add((MilitaryPlane) plane);
+            }
+        }
+        return militaryPlaneList;
+    }
+
+    public PassengerPlane getPassengerPlaneWithMaxPassengersCapacity() {
+        List<PassengerPlane> passengerPlanes = getPassengerPlanes();
+        PassengerPlane planeWithMaxCapacity = passengerPlanes.get(0);
+        for (int i = 0; i < passengerPlanes.size(); i++) {
+            if (passengerPlanes.get(i).getPassengersCapacity() > planeWithMaxCapacity.getPassengersCapacity()) {
+                planeWithMaxCapacity = passengerPlanes.get(i);
+            }
+        }
+        return planeWithMaxCapacity;
+    }
+
+    public List<MilitaryPlane> getTransportMilitaryPlanes() {
```

Comment:
I'd combine these 2 methods together to avoid duplication
like DRY
```
public List<MilitaryPlane> getTransportMilitaryPlanes(MillitaryType type)
```